drivers/sn32/rgb_matrix_sn32f248b.c

```diff
@@ -355,15 +324,15 @@ void SN32F24XX_init(void) {
 static void flush(void) {}
 
 void SN32F24XX_set_color(int index, uint8_t r, uint8_t g, uint8_t b) {
```

Comment:
Functions SN32F24XX_init(), SN32F24XX_set_color(), SN32F24XX_set_color_all() can be made static.
And should be removed from rgb_matrix_sn32f24xx.h
The SN32F24XX_ prefix is also no longer needed, and can be removed.